packages/survey-creator-core/src/components/tabs/logic-types.ts

```diff
@@ -105,6 +105,8 @@ export class SurveyLogicType {
     if ((<any>obj).setOwner) {
       (<any>obj).setOwner(this.survey);
     }
+    //Do not fire notification about changes to our creator.survey model
+    obj["isGhost"] = true;
```

Comment:
is it secure for our code?

Reply:
Yes, we have isGhost property from ItemValue and Triggers. 
We can move it to Base class. I am not sure we need it.

Thank you,
Andrew

Reply:
It looks like hack. I am affraid that somebody can add `isGhost` for any other object type in future... If we want to keep isGhost **only** in ItemValue and Trigger maybe it could be better if we add condition for this setting, something like...?
```
if(obj instance of ItemValue || obj instanceof Trigger) {
  obj["isGhost"] = true;
}
```
It still will be the hack, but more secure...